data/src/main/java/org/oppia/data/backends/gae/ModifyJsonInterceptor.kt

```diff
@@ -0,0 +1,34 @@
+package org.oppia.data.backends.gae
+
+import okhttp3.*
+
+import java.io.IOException
+
+//TODO: Transfer this XSSI_PREFIX to a constant file
```

Comment:
You might as well do this now, here and elsewhere? Try to avoid leaving TODOs behind you in the codebase.

Reply:
Actually these TODOs are just a small part of a larger PR that we will have to do at later stage. Thus cannot solve this right now.